jsonschema-core.xml

```diff
@@ -800,49 +800,291 @@
+                vocabularies to be combined in different ways, and for meta-schema authors
+                to impose additional constraints such as forbidding certain keywords, or
+                performing unusually strict syntactical validation, as might be done
+                during a development and testing cycle.
             </t>
+            <section title='The "$schema" Keyword'>
+                <t>
+                    The "$schema" keyword is both used as a JSON Schema feature set identifier and
+                    the location of a resource which is itself a JSON Schema, which describes any
+                    schema written for this particular feature set.
+                </t>
+                <t>
+                    The value of this keyword MUST be a <xref target="RFC3986">URI</xref>
+                    (containing a scheme) and this URI MUST be normalized.
+                    The current schema MUST be valid against the meta-schema identified by this URI.
+                </t>
+                <t>
+                    If this URI identifies a retrievable resource, that resource SHOULD be of
+                    media type "application/schema+json".
+                </t>
+                <t>
+                    The "$schema" keyword SHOULD be used in a root schema.
+                    It MUST NOT appear in subschemas.
+                </t>
+                <t>
+                    <cref>
+                        Using multiple "$schema" keywords in the same document would imply that the
+                        feature set and therefore behavior can change within a document.  This would
+                        necessitate resolving a number of implementation concerns that have not yet
+                        been clearly defined.  So, while the pattern of using "$schema" only in root
+                        schemas is likely to remain the best practice for schema authoring,
+                        implementation behavior is subject to be revised or liberalized in
+                        future drafts.
+                    </cref>
+                    <!--
+                        In particular, the process of validating an instance, including validating a
+                        schema as an instance against its meta-schema, only allows for a single set
+                        of rules across the entire instance document.  There is no equivalent of
+                        changing the meta-schema partway through the validation for non-schema
+                        instances.
+                    -->
+                </t>
+                <t>
+                    Values for this property are defined elsewhere in this and other documents,
+                    and by other parties.  JSON Schema implementations SHOULD implement support
+                    for current and previous published drafts of JSON Schema vocabularies
```

Comment:
Section "Meta-Schemas and Vocabularies" is really fine, and it describes very well the separation of roles between meta-schemas and vocabularies. However, I find a bit confusing indicating in this section (related to meta-schemas) something regarding implementation of vocabularies.

This paragraph is carried over almost without changes from draft-05, where meta-schemas and vocabularies were not differentiated, and I find this wording confusing in the new context.

I don't really understand if it is pretended that implementations should first try to recognize the "feature set" from the plain `$schema` value, then if it is not recognized as a fallback they should analyze the meta-schema and check the feature set from the meta-schema's `$vocabulary`; or the other way around.

Reply:
Ah, good catch, thanks!  I will clarify.

Reply:
@jgonzalezdr please let me know if the revised language is acceptable, and if so I will mark this thread as resolved.